src/vernemq_dev_topic.erl

```diff
@@ -0,0 +1,34 @@
+%% Copyright 2017 Erlio GmbH Basel Switzerland (http://erl.io)
```

Comment:
I'm curious why keep part of the functions here and the others in the VerneMQ core, any topic validation can be useful for plugin developers IMO and it doesn't hurt or expose harmful functionality.

Reply:
So far the only function I know that people are using is the `unword` function. We should add more functions, but if we start to give API guarantees it should be carefully thought out before doing so. We can't remove it afterwards.

So it's simply trying to be cautious.

Reply:
Thats very True too, I only use `unword` myself, not sure about others out there

Reply:
Guess we can't really know until someone complains or asks for something more... or we ourselves see a need for something.

Reply:
fair